tests/test_subscription.py

```diff
@@ -831,18 +830,17 @@ def test_run_transmogrifier_chained_subscription_exclude_site_algo(self, rse_fac
             rules = [rule for rule in rucio_client.list_did_rules(scope=tmp_scope.external, name=dsn) if str(rule['subscription_id']) == str(subid)]
             if rules[0]['source_replica_expression']:
                 rules.reverse()
-            print(rules, dict_rse)
-            assert len(rules) == 2
+            assert len(rules) == 2, f"Expected 2 rules, got {len(rules)}. Rules: {rules}, Dict RSE: {dict_rse}"
```

Comment:
nit: I'd call it
```suggestion
            assert len(rules) == 2, f"Expected 2 rules, got {len(rules)}. Rules: {rules}, RSE dict: {dict_rse}"
```

but it's not worth changing